src/forms/signpost/SignPost.tsx

```diff
@@ -14,6 +14,10 @@ import {Button} from "../button";
 import {Icon, IconProps} from "../../icon";
 import {calculateAxisPosition} from "./AxisPosition";
 
+//className constants
+const signPostActive = "signpost-action signpost-trigger active";
```

Comment:
Please create different constants for these 3 classes and use ClassNames() in utils to assign array of class names to div/button

Reply:
Done.